ParameterDataApi.yaml

```diff
@@ -0,0 +1,245 @@
+  contact:
+    email: matt.synborski@resilienx.com
+  version: 0.0.0
+externalDocs:
+  description: Find out more about Swagger
+  url: http://swagger.io
+
+servers:
+- url: https://<site>.fraihmwork.resilienx.com/api/parameter/v0
+
+tags:
+- name: Parameter
+  description: Parameter data model manipulation endpoints
+
+paths:
+  /parameter:
+    post:
+      tags:
+        - Parameter
+      summary: Add new time series parameter data to FRAIHMWORK model
+      description: Add new time series parameter data to FRAIHMWORK model
+      operationId: registerParameterData
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TimeSeriesDataDefinition'
+      responses:
+        201:
+          description: POSTed new time series parameter data
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+          
+        400:
+          description: Bad Request (invalid submission). 
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+          
+        401:
+          description: Not authenticated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        403:
+          description: Forbidden
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        406:
+          description: Not acceptable - only responds in application/json format
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        500:
+          description: Server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+      security:
+      - ParameterDataDefinitionsAuth:
+        - fraihmwork.parameter.data.write
+
+components:
+  schemas:
+    Response:
+      type: object
+      description: A default uuid / message combination to convey simple success / error messages to the client
+      properties:
+        itemId:
+          type: string
+          format: uuid
+        message:
+          type: string
+          description: The success message provided back to the client
+      required:
+        - message
+        
+    ColumnDefinition:
+      type: array
+      items:
+        $ref: '#/components/schemas/ColumnElementDefinition'
+          
+    ColumnElementDefinition:
+      type: object
+      description: The column elements      
+      properties:
+        id:
+          description: The identifier for the column
+          type: string
+          format: byte
```

Comment:
The format of 'byte' seems odd. Do we expect this to be base64 encoded data for the id? Does this ID field even matter if we name the column below anyway? How will a client posting TS data determine this id? This strikes me as an eagle.io internal-specific thing